app/views/info/about/_show_love_bristol.md

```diff
@@ -1,3 +1,3 @@
 ### Show some love
 
-If you like Swing Out London and find it useful, please consider linking to it from your Blog, Facebook, Twitter or personal website: this really helps Google to know that people are interested and will make it easier for others to find the site.
+If you like Swing Out Bristol and find it useful, please consider linking to it from your Blog, Facebook, Twitter or personal website: this really helps Google to know that people are interested and will make it easier for others to find the site.
```

Comment:
Maybe this text is generic enough that it should just be the same between the two pages, with just the site name changed?